app/assets/stylesheets/application.scss

```diff
@@ -26,3 +26,4 @@ $icon-font-path: "bootstrap-sass/assets/fonts/bootstrap/";
 @import "confirm_box";
 @import "inner-pages";
 @import "homepage";
+@import "request_account";
```

Comment:
I think the CSS file must also be renamed to `account_request` for consistency.

Reply:
So, we got a stylesheet named `account_requests.scss` when we generated the scaffold. We have added the formatting to this file and removed the `account_request`